sci-libs/proj/proj-5.2.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DATUMGRID="${PN}-datumgrid-1.8.tar.gz"
+
+DESCRIPTION="PROJ coordinate transformation software"
+HOMEPAGE="https://proj4.org/"
+SRC_URI="
+	http://download.osgeo.org/proj/${P}.tar.gz
+	http://download.osgeo.org/proj/${DATUMGRID}
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
+IUSE="static-libs"
+
+RDEPEND="${DEPEND}"
```

Comment:
This is rather useless, but LGTM otherwise! :)

Reply:
Removed. Didn't make much sense but it was in skel.ebuild until last week 7d0500007c.